src/menu/host_interface.c

```diff
@@ -111,6 +112,7 @@ void main_menu_host_interface(engine_ctx_t *ctx,
   switch (applet_id) {
     case 1: {
       manager_app_main(usb_evt);
+      main_menu_set_update_req(true);
```

Comment:
this should go after the switch to cover all the cases.

Reply:
added